commit 878a667394501e03cd27b393b87cf472fa74122c
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Thu Oct 7 16:28:15 2010 -0700

    Staging: brcm80211: remove unneeded pm macro
    
    Use the real type, not a macro.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/linuxver.h
+++ b/drivers/staging/brcm80211/include/linuxver.h
@@ -39,9 +39,6 @@ typedef irqreturn_t(*FN_ISR) (int irq, v
 #include <linux/sched.h>
 #include <linux/ieee80211.h>
 
-/* suspend args */
-#define DRV_SUSPEND_STATE_TYPE pm_message_t
-
 #define CHECKSUM_HW	CHECKSUM_PARTIAL
 
 #include <linux/time.h>
--- a/drivers/staging/brcm80211/sys/wl_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wl_mac80211.c
@@ -1359,7 +1359,7 @@ wl_pci_probe(struct pci_dev *pdev, const
 }
 
 #ifdef LINUXSTA_PS
-static int wl_suspend(struct pci_dev *pdev, DRV_SUSPEND_STATE_TYPE state)
+static int wl_suspend(struct pci_dev *pdev, pm_message_t state)
 {
 	wl_info_t *wl;
 	struct ieee80211_hw *hw;
